elide-core/src/main/java/com/yahoo/elide/resources/JsonApiEndpoint.java

```diff
@@ -112,13 +113,29 @@ public Response patch(
      */
     @DELETE
     @Path("{path:.*}")
+    @Consumes("application/vnd.api+json")
     public Response delete(
         @PathParam("path") String path,
         @Context SecurityContext securityContext,
         String jsonApiDocument) {
         return build(elide.delete(path, jsonApiDocument, getUser.apply(securityContext)));
     }
 
+    /**
+     * Delete resource handler (expects no body and no content-type header).
+     *
+     * @param path request path
+     * @param securityContext security context
+     * @return response
+     */
+    @DELETE
+    @Path("{path:.*}")
+    public Response deleteWithoutBody(
```

Comment:
@clayreimann Actually old `delete` method allows you delete resources too - not only relationships. The only difference is it requires mandatory content-type header and empty body in this case.

Also, I think it's very minor change - it doesn't  change existing behavior at all and just adds one more way to delete things. So I'm not sure if it deserves 3.0 branch.